### PR TITLE
Wait for deployments to complete before installing cluster versions

### PR DESCRIPTION
**Pull request by csrwng**

### CODE REVIEW

**Comment by csrwng** [comment]
@dgoodwin ptal

File: contrib/ansible/deploy-devel-playbook.yml [hunk 1]
```diff
@@ -16,6 +16,10 @@
     # Namespace to deploy CO to:
     cluster_operator_namespace: "openshift-cluster-operator"
   tasks:
+  - name: wait for apiserver and controller-manager deployment to finish
+    shell: |-
+      oc rollout status deployment/cluster-operator-apiserver -n {{ cluster_operator_namespace }}
+      oc rollout status deployment/cluster-operator-controller-manager -n {{ cluster_operator_namespace }}
```
Comment (dgoodwin) [inline contrib/ansible/deploy-devel-playbook.yml]: IIRC we're supposed to try to stick to command over shell as it's a bit safer particularly when you've got vars going in. (@abutcher correct me if I'm wrong)

Lets just do it as two separate tasks, more informative anyhow for someone watching it run.

**Comment by csrwng** [comment]
@dgoodwin addressed comments

File: contrib/ansible/deploy-devel-playbook.yml [hunk 2]
```diff
@@ -16,6 +16,13 @@
     # Namespace to deploy CO to:
     cluster_operator_namespace: "openshift-cluster-operator"
   tasks:
+  - name: wait for apiserver deployment to finish
+    command: |-
+      oc rollout status deploymentconfig/cluster-operator-apiserver -n {{ cluster_operator_namespace | quote }}
```
Comment (dgoodwin) [inline contrib/ansible/deploy-devel-playbook.yml]: TIL about | quote, cool!

**Pull request merged**